generator/committee_readme.tmpl

```diff
@@ -8,6 +8,9 @@ The [charter]({{.CharterLink}}) defines the scope and governance of the {{.Name}
 {{- if .Meetings }}
 
 ## Meetings
+{{- if .Contact.MailingList }}
+*Joining the [mailing list]({{.Contact.MailingList}}) for the group will typically add invites for the following meetings to your calendar.*
+{{- end }}
```

Comment:
```suggestion
## Meetings

{{- if .Contact.MailingList }}
*Joining the [mailing list]({{.Contact.MailingList}}) for the group will typically add invites for the following meetings to your calendar.*
{{- end }}

```

Adding some spacing to prevent a markdown lint warning.